### PR TITLE
Supress <unk>

### LINKED ISSUE #673: Add a suppress-unk option
During runtime, we sometimes don't want to output `<unk>` tokens.

### PR DESCRIPTION
**Pull request by jlibovicky**
Closes #673.

### CODE REVIEW

File: neuralmonkey/decoders/autoregressive.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def extend_namedtuple(name: str, parent: Type,
      ("prev_logits", tf.Tensor)])
 
 
-# pylint: disable=too-many-public-methods
+# pylint: disable=too-many-public-methods,too-many-arguments,too-many-instance-attributes
```
Comment (jindrahelcl) [inline neuralmonkey/decoders/autoregressive.py]: Too many arguments už je před konstruktorem

File: neuralmonkey/decoders/autoregressive.py [hunk 2]
```diff
@@ -189,9 +193,12 @@ def embedding_matrix(self) -> tf.Variable:
     def get_logits(self, state: tf.Tensor) -> tf.Tensor:
         """Project the decoder's output layer to logits over the vocabulary."""
         state = dropout(state, self.dropout_keep_prob, self.train_mode)
+        logits = tf.matmul(state, self.decoding_w) + self.decoding_b
 
-        logits = tf.matmul(state, self.decoding_w)
-        logits += self.decoding_b
+        if self.supress_unk:
+            one_hot = tf.one_hot(
```
Comment (jindrahelcl) [inline neuralmonkey/decoders/autoregressive.py]: Přejmenovat na unk_mask nebo tak něco

File: neuralmonkey/decoders/autoregressive.py [hunk 3]
```diff
@@ -189,9 +193,12 @@ def embedding_matrix(self) -> tf.Variable:
     def get_logits(self, state: tf.Tensor) -> tf.Tensor:
         """Project the decoder's output layer to logits over the vocabulary."""
         state = dropout(state, self.dropout_keep_prob, self.train_mode)
+        logits = tf.matmul(state, self.decoding_w) + self.decoding_b
 
-        logits = tf.matmul(state, self.decoding_w)
-        logits += self.decoding_b
+        if self.supress_unk:
+            unk_mask = tf.one_hot(
+                UNK_TOKEN_INDEX, depth=len(self.vocabulary), on_value=-1e-9)
```
Comment (varisd) [inline neuralmonkey/decoders/autoregressive.py]: Even though this is closed already.... Why is the on_value=-1e-9 ??? Shouldn't it be -1e9 ???
Now you are just lowering the logits of UNK_TOKENs by a very small number.
